aiverify-foundation/moonshot-data
# update the testing files for GSM8k and SQuAD2.0

## Description

update the testing files for GSM8k and SQuAD2.0 datasets.

## Motivation and Context

[Explain the motivation or the context behind this pull request. Why is it necessary?]

## Type of Change













## How to Test

For GSM8k, please run the command line: Run command line: run_recipe "my run gsm8k dataset push to github together llama3.1 405b v4" "['my-gsm8k']" "['my-together-meta-llama-3-1-405b-instruct-connector']" -n 10 -r 1 -s " "
Expected testing result:
![image](https://github.com/user-attachments/assets/dfc19bf5-8362-46cb-963f-b6465b6da59a)
For SQuAD2.0, please run the command line: run_recipe "my run squad v2 dataset push to github together llama3.1 405b v2" "['my-squad-v2']" "['my-together-meta-llama-3-1-405b-instruct-connector']" -n 10 -r 1 -s " "
Expected testing result:
![image](https://github.com/user-attachments/assets/4f17a053-9360-4e18-b4c5-d5366c425e9d)

Those are the test results for a subset with only 10 examples. If you need to test on the full dataset, please remove “-n 10” from the command line.

## Checklist

Please check all the boxes that apply to this pull request using "x":


## Screenshots (if applicable)

NA

## Additional Notes

NA

<details>
  <summary>Developer Certificate of Origin</summary>

 ```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
 ```
</details>
